include/adc.h

```diff
@@ -294,6 +319,26 @@ struct adc_driver_api {
 #endif
 };
 
+/**
+ * @brief Retrieve the internal reference voltage
+ *
+ * @param dev Pinter to the device structure for the driver instance.
+ *
+ * @return The internal reference, in millivolts.
+ * @retval -ENOTSUP if there is no internal reference
```

Comment:
I guess `@return` and `@retval` should not be used together. Wouldn't it be better to just add ", or -ENOTUP if ..." to the description above?

Reply:
It would be better if `@return` and `@retval` could be used together, so the explanation of how each error value should be interpreted could be provided.  However, we must deal with the world as it is, not as we wish it would be....

Done.